src/Metadata/IdentifiersExtractorInterface.php

```diff
@@ -28,6 +29,7 @@ interface IdentifiersExtractorInterface
      * @param array<string, mixed> $context
      *
      * @throws RuntimeException
+     * @throws InvalidArgumentException
```

Comment:
Catch somewhere, so we need to update the phpdoc to avoid a `catch.neverThrown` error